tensorflow/lite/micro/tools/generate_cc_arrays.py

```diff
@@ -90,7 +90,12 @@ def generate_array(input_fname):
 
 
 def get_array_name(input_fname):
-  base_array_name = 'g_' + input_fname.split('.')[0].split('/')[-1]
+  # Normalize potential relative path to remove additional dot.
+  abs_fname = os.path.abspath(input_fname)
+  print(abs_fname)
+  print(abs_fname.split('.')[0])
+  base_array_name = 'g_' + abs_fname.split('.')[-2].split('/')[-1]
+  print(base_array_name)
```

Comment:
I think we might want to remove some of these prints?

Reply:
absolutely!  Thanks for catching this!  This is silly of me. Note that internal cl does not have this issue. I imported a pre-clean up version by mistake.